### DISCUSSION TITLE
Gmail issue - "Be careful with this message. Gmail could not verify that it actually came from..."

### DISCUSSION THREAD

**Discussion by davecyen** [vercel/vercel#4554] [Help]
I've transferred my domain from Google Domains to Vercel, however I'm experiencing issues when sending email in Gmail. I receive the following error message when a team member from my domain sends me an email:

"Be careful with this message. Gmail could not verify that it actually came from <domain name>. Avoid clicking links, downloading attachments, or replying with personal information."

I've also experienced issues with some external companies being unable to email us. How do I fix this?

**Comment by mcsdevv** [comment]
Hello @davecyen, can you provide a little more detail on the steps you've taken with the setup here? Have you transferred the domain entirely or completed a nameserver change?

What steps have you taken after this and what records have you added?
